tests/contrib/test_multiprocess_file_storage.py

```diff
@@ -0,0 +1,293 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for oauth2client.multistore_file."""
+
+import contextlib
+import datetime
+import json
+import multiprocessing
+import os
+import tempfile
+import unittest2
+import fasteners
+import mock
+
+from oauth2client.client import OAuth2Credentials
+from oauth2client.contrib import multiprocess_file_storage
+from six import StringIO
+
+from ..http_mock import HttpMockSequence
+
+_filehandle, FILENAME = tempfile.mkstemp('oauth2client_test.data')
+os.close(_filehandle)
+
+
+@contextlib.contextmanager
+def scoped_child_process(target, **kwargs):
+    die_event = multiprocessing.Event()
+    ready_event = multiprocessing.Event()
+    process = multiprocessing.Process(
+        target=target, args=(die_event, ready_event), kwargs=kwargs)
+    process.start()
+    try:
+        ready_event.wait()
+        yield
+    finally:
+        die_event.set()
+        process.join(5)
+
+
+def _create_test_credentials(expiration=None):
+    access_token = 'foo'
+    client_secret = 'cOuDdkfjxxnv+'
+    refresh_token = '1/0/a.df219fjls0'
+    token_expiry = expiration or (
+        datetime.datetime.utcnow() + datetime.timedelta(seconds=3600))
+    token_uri = 'https://www.google.com/accounts/o8/oauth2/token'
+    user_agent = 'refresh_checker/1.0'
+
+    credentials = OAuth2Credentials(
+        access_token, 'test-client-id', client_secret,
+        refresh_token, token_expiry, token_uri,
+        user_agent)
+    return credentials
+
+
+class MultiprocessStorageBehaviorTests(unittest2.TestCase):
+
+    def setUp(self):
+        try:
+            os.unlink(FILENAME)
+            os.unlink('{}.lock'.format(FILENAME))
+        except OSError:  # pragma: NO COVER
+            pass
+
+    def tearDown(self):
+        try:
+            os.unlink(FILENAME)
+            os.unlink('{}.lock'.format(FILENAME))
+        except OSError:  # pragma: NO COVER
+            pass
+
+    def test_basic_operations(self):
+        credentials = _create_test_credentials()
+
+        store = multiprocess_file_storage.MultiprocessFileStorage(
+            FILENAME, 'basic')
+
+        # Save credentials
+        store.put(credentials)
+        credentials = store.get()
+
+        self.assertNotEquals(None, credentials)
```

Comment:
`assertNotNone`, please? You've already imported `unittest2`, so Python 2.6 compatibility doesn't count as a reason to use the less-specific assertion method.